test/integration/features/cmd-docker-env.feature

```diff
@@ -0,0 +1,38 @@
+@cmd-docker-env @command
+Feature: Command docker-env
+Command docker-env sets docker environment variables for supported shells.
+INFO: This feature runs against shell instance. To use non-default shell, please select
+one from: bash, cmd, powershell, tcsh, zsh with TEST_WITH_SHELL parameter of make integration.
+
+  Scenario: Docker client is not set up
+  INFO: This scenario starts interactive shell instance, which will be closed in the end of this feature.
+    Given user starts shell instance
+     When executing in shell "docker info"
```

Comment:
I think we need to be more specific
```cucumber
   When user start local shell instance
```

to prevent the confusion of where the command `docker info` is executed.
Especially because of "Name: minishift" in the same scenario, it can confuse

Reply:
or sth like `host`, as:

```cucumber
  When executing in local shell "docker info"
```

Reply:
@gbraad Originally I used "persistent shell" to highlight that in contrary to `executing "minishift/oc <command> <subcommand>"` the changes will remain - for example the exports.

I am fine with mentioning it as `local shell`.